frontend/src/utils/api.ts

```diff
@@ -0,0 +1,11 @@
+export async function downloadFile(){
+    const token = JSON.parse(localStorage.getItem('JWT')!).accessToken;
+
+    return fetch('http://localhost:5000/users/calendar', {
```

Comment:
jak zrobie deploya to mi localhost nie bedzie dzialac, uzyj `BASE_PATH` z base.ts